source/community.rst

```diff
@@ -0,0 +1,16 @@
+At the time of writing, accessibility is not a regular consideration in many 
+Scientific Python projects. Many projects may have a handful of people who 
+have accessibility knowledge or are interested in accquiring more, but may not 
+be able to reach critical mass on a per-project basis. Connecting Scientific 
+Python community members focused on accessibility will provide a boon for 
+individuals, projects, and the ecosystem at large. 
+
+Working with the community may include activities that
+
+* Make the ecosystem more accessible 
+* Clarify the broad term accessibility and its variouspossible implementations
+* Provide specific assessment tools and steps
+* Direct to existing accessibility standards and modifications for open source 
+    specifically 
+* Provide case studies of how different core projects became more accessible
+* Start discussion and develop shared solutions
```

Comment:
In this area, we could add
- What platforms we meet and discuss this work
  - So far, that is at the accessibility.scientific-python.org repo, the SPECs repo and Discord
 - How to get involved in this community
   - Email us, post a Github issue in accessibility.scientific-python.org repo
- How people can add future recommendations
- The previous alt-text workshops we have hosted
- Who is currently in the community (Isabela, Mars)